tailwind.config.js

```diff
@@ -13,20 +14,25 @@ module.exports = {
         'light-wool': "url('/assets/light-wool.png')"
       })
     },
-    // colors: {
-    //   black: gruvbox.dark.hard,
-    //   white: colors.white,
-    //   paper: gruvbox.light[0],
-    //   primary: gruvbox.dark[0],
-    //   secondary: gruvbox.red.neutral,
-    //   ...gruvbox,
-    // },
+    colors: {
+      ...colors,
+      apnaGreen: "#04D98B",
+      dateGreen: "#038C5A",
+      apniHeading: "#014029",
+      neechuBG: "#3B3C40",
+      neechuText: "#f2f2f2",
+      ivory: "#FFFFF0",
+    },
     fontFamily: {
       sans: [
         "system-ui",
         "sans-serif",
       ],
     },
+    boxShadow: {
+      ...defaultTailwind.boxShadow,
+      apnaShadow: "0 25px 50px -12px rgba(1, 64, 41, 0.25)"
+    }
```

Comment:
Please use a more semantic name for custom shadows.

Reply:
https://github.com/ILUGD/planet/pull/17/commits/9343d8e163eec3d570f14df551aea0260a214137 - fixed here